### ISSUE TITLE
The pre-trained model?

### ISSUE THREAD

**Issue by PureMmm** [IceClear/MW-GAN#10]
Hello, I would like to ask you a question. I saw in your paper that you trained data on QP32 and then tested data on QP37 and QP32, but what you provided on GitHub is the model of QP42 (which can be seen from the name of the model).So the model you provided is QP42, not 32, right?
thank!

**Comment by IceClear** [comment]
Thanks for your interest. Yes, as introduced in readme, this code is the enhanced version of the conference version and the implement of the journel paper where we mainly test on QP42 (in submittion). If you want to test on other QPs, I recommend you to finetune on the provided model for time saving.

**Comment by PureMmm** [comment]
thank for your reply,When I tested your QP42 model on the QP37 test set, its LPIPS was very bad, not lower than the LPIPS of the input frame, but higher?Is that normal?Is this because the OP42 model does not match the 37 model

**Comment by IceClear** [comment]
Yes. QP42 frames are very different from QP37. Thus, if you want to obtain similar results reported in the paper, finetuning or retraining the model is necessary. However, metric results such as LPIPS usually cannot directly reflect the real perceptual quality as humen feel. If you observe the results, they may not be as bad as what LPIPS indicates.

**Comment by PureMmm** [comment]
thank you very much!So far I have seen some detailed textures generated by GAN, If I have time, I will fine-tune model for further testing

**Comment by PureMmm** [comment]
Could you share the model or enhanced pngs on QP37 in your paper?Thank you very much!

**Comment by IceClear** [comment]
Sorry, currently I am busy with other projects and the data related to this project is really a mess since I used to conduct lots of experiments. Thus it is not easy to find the related model for you in a short time. Anyway, just as I said, finetuning a new model follow the readme is not difficult and I recommend you to have a try.

**Comment by PureMmm** [comment]
okkkkk,I would try!thank for your reply again!
